src/gui/addnewtorrentdialog.cpp

```diff
@@ -369,7 +384,7 @@ void AddNewTorrentDialog::saveSavePathHistory() const
         // Add save path to history
         history.push_front(selectedSavePath.absolutePath());
         // Limit list size
-        if (history.size() > 8)
+        if (history.size() > savePathHistoryLength())
```

Comment:
It would be nice to increase `maxVisibleItems` of `savePathComboBox` (addnewtorrentdialog.ui). Otherwise it's hard to scroll through like 100 or so paths via a "window" of 10 (default) (http://doc.qt.io/qt-5/qcombobox.html#maxVisibleItems-prop)

I'm using a custom build with visible items increased to 20 and it is much better (personal opinion) than default 10.
Here the reference patch: [link](https://github.com/d-s-x/qBittorrent/commit/f87cda59a0e90717ac5f6f92d3e339881278d086?diff=unified#diff-c076510ade01422e9203c8dfd8bd27f6)

Reply:
Thanks, done.